.eslintrc.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
     "env": {
+        "es6": true,
```

Comment:
es6 shouldn't be part of this.

Reply:
I'll remove it. However, you are targeting Node 6.x+ according to [README](https://github.com/nats-io/node-nats#supported-node-versions). Without it linter complains about every ES6 feature that's part of Node 6.x. Should this be in a separate PR then?